.github/fabricbot.json

```diff
@@ -0,0 +1,106 @@
+[
+    {
+      "taskType": "trigger",
```

Comment:
Any reason you kept the PR autoclose code in fabric bot? We have not used that in the past in the SDK.

Reply:
Hi @marcpopMSFT , I just replied you offline :)